File: go/muse/algorithms/meta/Sorting/Sorting.go
package Sorting

import (
	bubble_sort "muse/algorithms/sorting/BubbleSort"
	insertion_sort "muse/algorithms/sorting/InsertionSort"
	merge_sort "muse/algorithms/sorting/MergeSort"
	quick_sort "muse/algorithms/sorting/QuickSort"
	selection_sort "muse/algorithms/sorting/SelectionSort"
)

func BubbleSort(arr []int) {
	bubble_sort.Sort(arr)
}

func InsertionSort(arr []int) {
	insertion_sort.Sort(arr)
}

func MergeSort(arr []int) {
	merge_sort.Sort(arr)
}

func QuickSort(arr []int) {
	quick_sort.Sort(arr)
}

func SelectionSort(arr []int) {
	selection_sort.Sort(arr)
}

File: archive/nodejs/muse/algorithms/Sorting.js
import * as BubbleSort from './sorting/BubbleSort.js';
import * as InsertionSort from './sorting/InsertionSort.js';
import * as MergeSort from './sorting/MergeSort.js';
import * as QuickSort from './sorting/QuickSort.js';
import * as SelectionSort from './sorting/SelectionSort.js';

export const bubbleSort = (arr) => {
  BubbleSort.sort(arr);
};

export const insertionSort = (arr) => {
  InsertionSort.sort(arr);
};

export const mergeSort = (arr) => {
  MergeSort.sort(arr);
};

export const quickSort = (arr) => {
  QuickSort.sort(arr);
};

export const selectionSort = (arr) => {
  SelectionSort.sort(arr);
};
